hack/prom-rule-ci/prom-rules-tests.yaml

```diff
@@ -249,111 +249,126 @@ tests:
               kubernetes_operator_part_of: "kubevirt"
               kubernetes_operator_component: "kubevirt"
 
-  # High REST errors
+  # Burst REST errors
   - interval: 1m
     input_series:
       - series: 'rest_client_requests_total{namespace="ci", pod="virt-controller-1", code="200"}'
-        values: "2 2 2 2 2 2 2 2 2 2"
+        values: '0+10x20'
```

Comment:
@Barakmor1 I see that you have increased the amount from 2 to 20 but it's unclear to me why, maybe you can elaborate about this change in the commit message?

Reply:
Hey Igor ,
to be honest i changed the entire test because it simulated a case where alert shouldn't be fire hence it fail with correct logic.
 i will try to explain:
this is one of the current test use-cases :
```
         - interval: 1m
            input_series:
       ....
       - series: 'rest_client_requests_total{namespace="ci", pod="virt-operator-1", code="200"}'
        values: "2 2 2 2 2 2 2 2 2 2"
      - series: 'rest_client_requests_total{namespace="ci", pod="virt-operator-1", code="400"}'
        values: "10 10 10 10 10 10 10 10 10 20"
        ...
        - eval_time: 5m
        alertname: VirtOperatorRESTErrorsBurst
        exp_alerts:
          - exp_annotations:
              summary: "More than 80% of the rest calls failed in virt-operator for the last 5 minutes"
              runbook_url: "https://kubevirt.io/monitoring/runbooks/VirtOperatorRESTErrorsBurst"
            exp_labels:
              pod: "virt-operator-1"
              severity: "critical"
          ...
```
currently the test run only 5 minutes so the number of New `rest_client_requests_total{namespace="ci", pod="virt-operator-1", code="200"}` is 0 because it is always 2(no new requests)
and the number of new `rest_client_requests_total{namespace="ci", pod="virt-operator-1", code="400"}`
is also 0 because the test run for only 5 minutes.
so our alert condition should be 0/0 > 0.8 which should NOT fire an alert
actually when the test run for only five minutes it will never fire an alert because of the `For:   5m` in the alert configuration